app/forms/hyrax/una_exhibition_form.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+module Hyrax
+  class UnaExhibitionForm < Hyrax::Forms::WorkForm
+    include ::HykuAddons::WorkForm
+
+    self.model_class = ::UnaExhibition
+    add_terms %i[title alt_title resource_type creator current_he_institution
+                 alt_email abstract keyword subject subject_text official_link
+                 alternate_identifier citation funder project_name fndr_project_ref
+                 event_title event_location event_date related_exhibition
+                 related_exhibition_venue related_exhibition_date
+                 license rights_holder rights_statement rights_statement_text contributor
+                 medium extent duration is_format_of language location longitude
+                 latitude georeferenced time refereed irb_number irb_status add_info]
+    self.terms -= %i[source]
+    self.required_fields = %i[title resource_type creator]
+    include Hyrax::DOI::DOIFormBehavior
```

Comment:
In UnaImageForm you have a line after `self.required_fields` which adds a little space to seperate the different sets of functionality. Can we also have that here.